picard/plugin.py

```diff
@@ -217,23 +271,94 @@ def load_plugin(self, name, plugindir):
             info[0].close()
         return plugin
 
-    def install_plugin(self, path, dest):
-        plugin_name = _plugin_name_from_path(path)
-        if plugin_name:
-            try:
-                dest_exists = os.path.exists(dest)
-                same_file = os_path_samefile(path, dest) if dest_exists else False
-                if os.path.isfile(path) and not (dest_exists and same_file):
-                    shutil.copy(path, dest)
-                elif os.path.isdir(path) and not same_file:
-                    if dest_exists:
-                        shutil.rmtree(dest)
-                    shutil.copytree(path, dest)
-                plugin = self.load_plugin(plugin_name, USER_PLUGIN_DIR)
-                if plugin is not None:
-                    self.plugin_installed.emit(plugin, False)
-            except (OSError, IOError):
-                log.warning("Unable to copy %s to plugin folder %s" % (path, USER_PLUGIN_DIR))
+    def install_plugin(self, path, overwrite_confirm=None, plugin_dir=USER_PLUGIN_DIR):
```

Comment:
> Do we expect anyone to use a different plugin_dir here in the near future?

I guess not, i suppose we could use `USER_PLUGIN_DIR` directly and drop this parameter.